### ISSUE TITLE
Hard questions

### ISSUE THREAD

**Issue by janseta** [xoseperez/m5stack-rfm95#1]
Hi,

Is possible to get someninfonabou the hard needed?

I mean some tech info about the hard to kwon where to get the hard:
the antenna, the conector..

**Comment by xoseperez** [comment]
Do you mean the Bill of Materials? Sure, I should have addee it already.

**Comment by janseta** [comment]
Yes, I mean the tech specs of the materials needed..  
a Billl would be great!

Another sugestion:
if you just made a HOLE on the PCB , you could atach a battery in the conector that the M5Stack has in the core itself.. 
In that case, you will not need the battery module...

**Comment by xoseperez** [comment]
Yes, I've also noticed that it could be convenient to have access to the battery connector in the Core.
